github/enterprise_manage_ghes.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+)
+
+// NodeQueryOptions specifies the optional parameters to the EnterpriseService
+// Node management APIS.
+type NodeQueryOptions struct {
+	// UUID filters issues based on the node UUID.
+	UUID string `url:"uuid,omitempty"`
```

Comment:
Everywhere you use `omitempty`, please make sure that the type is a reference type, so this should be "string" => "*string".